### ISSUE TITLE
My CPU TensorFlow binary was not compiled to use: AVX AVX2

### ISSUE THREAD

**Issue by haozhou00** [rstudio/tensorflow#411]
I am a newbie of tensorflow . I do not have NVIDIA GPU after I install tensorflow and run 

model <- keras_model_sequential() 
model %>% 
  layer_dense(units = 256, activation = 'relu', input_shape = c(784)) %>% 
  layer_dropout(rate = 0.4) %>% 
  layer_dense(units = 128, activation = 'relu') %>%
  layer_dropout(rate = 0.3) %>%
  layer_dense(units = 10, activation = 'softmax')

The error message is "I tensorflow/core/platform/cpu_feature_guard.cc:142] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2".

It seems the problem is caused by my poor CPU? what should I do on this computer? I installed it on another laptop and everything works well. 

Many thanks!

**Comment by skeydan** [comment]
Don't worry, that's just a harmless warning :-)
It means you might perhaps get better performance if building TensorFlow from source , but it is not at all comparable to the difference between running on CPU versus on a (low-end even) GPU ...

**Comment by haozhou00** [comment]
Many many thanks!  Yes it is just a harmless warning - I have spent a whole day to install all the combinations which I can imagine to get it out

**Comment by wlandau** [comment]
The messages are harmless, but they are still are distracting, which matters in pedagogical settings. I was able to suppress them with `Sys.setenv(TF_CPP_MIN_LOG_LEVEL = "1")`.
